notebooks/ch17/classification_multiclass.md

```diff
@@ -0,0 +1,312 @@
+
+
+```python
+# HIDDEN
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn.datasets import load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
```

Comment:
You can add:

```
pd.options.display.max_rows = 7
pd.options.display.max_columns = 8
pd.set_option('precision', 2)
```

Here to make DFs only display 2 numbers after the decimal and have better DF displaying.

(See https://github.com/DS-100/textbook/blob/master/starter.ipynb for the starter code I recommend.)